### PR TITLE
new page: mset9-android

### LINKED ISSUE #2397: Idea: Guide to install MSET9 via Android (partially tested)
I have made a guide on how to install MSET9 using Android

[mset9 installer android.txt](https://github.com/hacks-guide/Guide_3DS/files/13370502/mset9.installer.android.txt)

Im not sure if something is off or not, i have tested this whole guide but a problem occured at the end, with the 3DS not loading up the exploit to boot into SafeB9SInstaller

My theory is, my Android device is running a version below 5.0, if anyone is going to add this to the guide, make sure to test it first, since i am unable to

### PR DESCRIPTION
**Pull request by lilyuwuu**
What it says on the tin: MSET9 instructions for Android (and by extension ChromeOS) via Play Store, using @danny8376's [MSET9Installer4Android](https://github.com/danny8376/MSET9Installer4Android) and the unarchive/file browser tool [ZArchiver](http://zdevs.ru/en/), with screenshots. Resolves #2397, and also clears pathway for merging #2393.

Works on my machine, not responsible for any [fires](https://github.com/hacks-guide/Guide_3DS/assets/27717444/69bd85be-0b06-4722-9bde-3d5415240da6) that may occur.

Notes:
- Are there any things that we can/should commonize between MSET9 and MSET9 Play Store?
- Does #2393 properly account for this PR?
- Do these instructions work well *enough* for people on older versions of Android?
- Are there any other file managers we can/should use? (I don't really want to re-take these screenshots though lol)
- Troubleshooting wording differs slightly, we might want to account for the different error messages (with the same troubleshooting) on our troubleshooting.

### CODE REVIEW

File: images/screenshots/mset9/Screenshot_20231124_070435_ZArchiver.png
Comment (lifehackerhansol) [inline images/screenshots/mset9/Screenshot_20231124_070435_ZArchiver.png:1]: This file could be named differently

**Review by lifehackerhansol — COMMENTED** [review]
can't really think of any way to commonize these two due to the steps being vastly different between CLI and APK

landing page may be a good idea

the other PR has been merged so we can make any edits if need be to account for it (which should be the landing page)

new troubleshooting section maybe? (start including troubleshooting too?)

File: images/screenshots/mset9/Screenshot_20231124_070435_ZArchiver.png
Comment (lilyuwuu) [reply images/screenshots/mset9/Screenshot_20231124_070435_ZArchiver.png:1]: this file shouldn't even be here LOL

**Comment by eip618** [comment]
https://media.licdn.com/dms/image/D5612AQEQXjwvaM_8dg/article-cover_image-shrink_600_2000/0/1685605453893?e=2147483647&v=beta&t=i8pMT1i72nySJ-SaIYoKoNLHcH1G4gyA-XZoHVrMyE8

**Pull request merged**